test/addons-napi/test_instanceof/test.js

```diff
@@ -74,7 +74,8 @@ if (typeof Symbol !== 'undefined' && 'hasInstance' in Symbol &&
   compareToNative(x, MySubClass);
   compareToNative(y, MySubClass);
   compareToNative(x, MyClass);
-  compareToNative(y, MyClass);
+  // TODO: https://github.com/nodejs/abi-stable-node/issues/236
```

Comment:
Probably not, because these issues are specific to N-API on Node-ChakraCore. The tests pass with N-API on Node-V8.